types/beacon_types.go

```diff
@@ -167,7 +169,7 @@ func (n BeaconNetwork) EstimatedSlotAtTime(time int64) spec.Slot {
 }
 
 func (n BeaconNetwork) EstimatedTimeAtSlot(slot spec.Slot) int64 {
-	d := int64(slot) * int64(n.SlotsPerEpoch())
+	d := int64(slot) * int64(n.SlotDurationSec().Seconds())
```

Comment:
logic seems different, was this function used anywhere else? does is still work for the other places if true?

Reply:
it's used only in `calculateValidatorRegistration` to calculate a timestamp

Reply:
so this is a bug fix, great!